dbtmetabase/_exposures.py

```diff
@@ -360,6 +373,10 @@ def __format_exposure(
                 "email": creator_email,
             },
             "depends_on": list(depends_on),
+            "meta": {
+                "avg_query_time": average_query_time,
+                "last_used_at_utc": last_used_at_utc or 'n.a.'
```

Comment:
Same here, if not provided, shouldn't be included at all (build up `meta` conditionally as a dictionary variable and only set it when not empty)